ansibullbot/utils/shippable_api.py

```diff
@@ -21,8 +21,10 @@
 import requests
 from tenacity import retry, stop_after_attempt, wait_fixed, RetryError, TryAgain
 
-
-ANSIBLE_PROJECT_ID = u'573f79d02a8192902e20e34b'
+# FIXME: Need to get all of the Shippable Project IDs for the various collections and define them here
+# 5e664a167c32620006c9fa50 community.general
+#ANSIBLE_PROJECT_ID = u'573f79d02a8192902e20e34b'
+ANSIBLE_PROJECT_ID = u'5e664a167c32620006c9fa50' # community.general
```

Comment:
This should use the repo data structure (or config file), rather than be hardcoded